R/credentials_gce.R

```diff
@@ -74,36 +74,61 @@
 #' }
 credentials_gce <- function(scopes = "https://www.googleapis.com/auth/cloud-platform",
                             service_account = "default", ...) {
-  gargle_debug("trying {.fun credentials_gce}")
+  gargle_debug("Trying {.fun credentials_gce} ...")
   if (!is_gce()) {
+    gargle_debug(c("x" = "We don't seem to be on GCE."))
     return(NULL)
   }
 
-  gce_token <- fetch_gce_access_token(scopes, service_account = service_account)
+  scopes <- scopes %||% "https://www.googleapis.com/auth/cloud-platform"
```

Comment:
`token_fetch()` sets `scopes = NULL` by default, so this just reinforces the strongly preferred default in the GCE setting.